TerminalDocs/troubleshooting.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Windows Terminal Troubleshooting
+description: Learn fixes to common obstacles in the Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 05/19/2020
+ms.topic: troubleshooting
+ms.service: terminal
+---
+
+# Troubleshooting in Windows Terminal
```

Comment:
@DHowett-MSFT @zadjii-msft You two are the ones that most closely watch the repo. Any scenarios you see that are missing?